.github/workflows/ci.yml

```diff
@@ -215,10 +212,6 @@ jobs:
           CPPSTD: c++11
         - SWIGLANG: scilab
           CPPSTD: c++11
-          os: ubuntu-18.04
-        - SWIGLANG: scilab
-          CPPSTD: c++11
-          os: ubuntu-20.04
```

Comment:
After this change it looks like we're only testing the scilab version in `ubuntu-20.04`, which is 6.1.0.  Before we were testing 6.0.1 from `ubuntu-18.04` too.

Reply:
You are correct, after looking at other languages, I removed `os` selection. IMHO it will be more maintainable in the future to avoid specific checking (as the code is now handling both version).

Should I explicitly check for 6.0.1 on ubuntu-18.04 to pass this commit and remove it later ?

Reply:
I will add scilab 5.5 from scilab.org as the ci-linux-install.sh script has been updated to handle that.